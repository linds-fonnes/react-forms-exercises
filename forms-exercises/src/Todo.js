import React from "react";

const Todo = ({ item, removeItem }) => {
  return (
    <li id={item.id}>
      {item.name} <button onClick={() => removeItem(item.id)}>x</button>
    </li>
  );
};

export default Todo;
