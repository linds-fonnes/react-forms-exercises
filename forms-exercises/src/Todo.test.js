import React from "react";
import { render } from "@testing-library/react";
import Todo from "./Todo";

const item = { id: 9999, name: "walk dog" };
it("renders without crashing", function () {
  render(<Todo item={item} />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<Todo item={item} />);
  expect(asFragment()).toMatchSnapshot();
});
