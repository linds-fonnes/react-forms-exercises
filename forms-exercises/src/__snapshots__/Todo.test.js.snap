// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <li
    id="9999"
  >
    walk dog 
    <button>
      x
    </button>
  </li>
</DocumentFragment>
`;
